usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -54,24 +80,37 @@ def add_arguments(self, parser):
             default=CONFIG.SPARK_S3_BUCKET,
             help="The destination bucket in S3 for creating the tables.",
         )
+        parser.add_argument(
+            "--no-initial-copy",
+            action="store_true",
+            required=False,
+            help="Whether to skip copying tables from the 'raw' database to the 'int' database during initial_run.",
+        )
 
     def handle(self, *args, **options):
         with self.prepare_spark():
             self.etl_level = options["etl_level"]
             self.spark_s3_bucket = options["spark_s3_bucket"]
+            self.no_initial_copy = options["no_initial_copy"]
+
+            # Capture minimum last load date of the source tables to update the "last_load_date" after completion
+            last_procure_load = get_last_load_date("source_procurement_transaction")
+            last_assist_load = get_last_load_date("source_assistance_transaction")
```

Comment:
We have a method called `get_earliest_load_date()` in `last_load_date.py` that accepts a list of keys and returns the earliest load date. This can simplify this block of code.

Reply:
I took a look at that code.  Unfortunately, I think there is a bug in that code.  For example, if I were to call `get_earliest_load_date(("source_procurement_transaction", "source_assistance_transaction"), datetime(1970, 1, 1))`, then it appears that the code would only return a date corresponding to the earliest last load date of `source_procurement_transaction` or `source_assistance_transaction` if that date were earlier than 1970-01-01, but I don't think that's what we want that call to do.

Reply:
Assuming you agree with my assessment, do you want me to fix the code so that it only returns the default value if no last load date is found for any of the keys?

Reply:
Yes, there is a bug there. Good catch. I would say yes, that would be a good fix

Reply:
Done